.gitignore

```diff
@@ -1,7 +1,3 @@
-# Remove Cargo.lock from gitignore if creating an executable, leave it for libraries
-# More information here https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html
-Cargo.lock
```

Comment:
Why is the lock being removed?

Reply:
@simbleau All Linebender crates (apart from stragglers like this one) have a `Cargo.lock` committed to git. The Rust team changed their guidance.

Also, see https://github.com/linebender/rfcs/blob/main/rfcs/0005-version-matrix.md